index.js

```diff
@@ -354,20 +354,21 @@ program
   });
 
 /**
- * Creates a new script project. The project title is optional.
+ * Creates a new script project. The project title is optional. The project parent Id is optional.
```

Comment:
Now that we have multiple parameters, can you describe them roughly like this?

```js
/**
 * Creates a new script project.
 * @param {string} [scriptTitle] An optional project title.
 * @param {string} [scriptParentId] An optional project parent Id. The Drive ID of a parent file
 *   that the created script project is bound to. This is usually the ID of a
 *   Google Doc, Google Sheet, Google Form, or Google Slides file. If not set, a
 *   standalone script project is created.
 * @see https://developers.google.com/apps-script/api/reference/rest/v1/projects/create#body.request_body.FIELDS.parent_id
 */
```

Reply:
Sure, I will do it. Thanks